src/app.config.json

```diff
@@ -0,0 +1,69 @@
+{
+    "$schema": "../node_modules/@alfresco/adf-core/app.config.schema.json",
+    "ecmHost": "http://localhost",
+    "bpmHost": "http://localhost",
+    "backendHost": "http://localhost:8080",
+    
+    "providers" : "ALL",
+    "application": {
+        "name": "Alfresco ADF Appplication"
```

Comment:
Do we really want to stick with "Alfresco ADF Application" as our hello-world app name?

Reply:
Forgot to change it. Thank you!